packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,72 @@
+# Specify analysis options.
+
+# This file is the .analysis_options file used by Flutter editors, such as
+# Atom. It is very similar to flutter_tools/flutter_analysis_options; the only
+# difference (currently) is the public_member_api_docs option, which triggers
+# too many messages to be used in editors.
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
```

Comment:
i forget what strict call checks are.
we need super mixins for the framework to function.